File: java/Samples/BiometricStandards/CS/FCRecordSampleCS/AddFaceImageForm.cs
using System;
using System.Windows.Forms;
using Neurotec.Biometrics.Standards;

namespace Neurotec.Samples
{
	public partial class AddFaceImageForm : Form
	{
		public AddFaceImageForm()
		{
			InitializeComponent();

			string[] items = Enum.GetNames(typeof(FcrFaceImageType));
			cbFaceImageType.Items.AddRange(items);
			cbImageDataType.Items.AddRange(Enum.GetNames(typeof(FcrImageDataType)));
			cbFaceImageType.SelectedIndex = 0;
			cbImageDataType.SelectedIndex = 0;
		}

		public FcrFaceImageType FaceImageType
		{
			get
			{
				return (FcrFaceImageType)Enum.Parse(typeof(FcrFaceImageType), cbFaceImageType.SelectedItem.ToString());
			}
			set
			{
				cbFaceImageType.SelectedItem = Enum.GetName(typeof(FcrFaceImage), value);
			}
		}

		public FcrImageDataType ImageDataType
		{
			get
			{
				return (FcrImageDataType)Enum.Parse(typeof(FcrImageDataType), cbImageDataType.SelectedItem.ToString());
			}
			set
			{
				cbImageDataType.SelectedItem = Enum.GetName(typeof(FcrImageDataType), value);
			}
		}
	}
}

File: java/Samples/BiometricStandards/VB/FCRecordSampleVB/AddFaceImageForm.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Windows.Forms
Imports Neurotec.Biometrics.Standards

Partial Public Class AddFaceImageForm
	Inherits Form
	Public Sub New()
		InitializeComponent()

		Dim items() As String = System.Enum.GetNames(GetType(FcrFaceImageType))
		cbFaceImageType.Items.AddRange(items)
		cbImageDataType.Items.AddRange(System.Enum.GetNames(GetType(FcrImageDataType)))
		cbFaceImageType.SelectedIndex = 0
		cbImageDataType.SelectedIndex = 0
	End Sub

	Public Property FaceImageType() As FcrFaceImageType
		Get
			Return CType(System.Enum.Parse(GetType(FcrFaceImageType), cbFaceImageType.SelectedItem.ToString()), FcrFaceImageType)
		End Get
		Set(ByVal value As FcrFaceImageType)
			cbFaceImageType.SelectedItem = System.Enum.GetName(GetType(FcrFaceImage), value)
		End Set
	End Property

	Public Property ImageDataType() As FcrImageDataType
		Get
			Return CType(System.Enum.Parse(GetType(FcrImageDataType), cbImageDataType.SelectedItem.ToString()), FcrImageDataType)
		End Get
		Set(ByVal value As FcrImageDataType)
			cbImageDataType.SelectedItem = System.Enum.GetName(GetType(FcrImageDataType), value)
		End Set
	End Property
End Class
